Directory.Packages.props

```diff
@@ -23,7 +23,7 @@
     <PackageVersion Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0" />
     <PackageVersion Include="Microsoft.AspNetCore.Hosting" Version="2.2.7" />
     <PackageVersion Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.2.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.JsonPatch" Version="8.0.0-preview.2.23153.2" />
+    <PackageVersion Include="Microsoft.AspNetCore.JsonPatch" Version="6.0.21" />
```

Comment:
Update to proper 8.0 version here and also in the inner api

Reply:
Done